src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -248,6 +320,28 @@ export class InstructorCoursesPageComponent implements OnInit {
     }, () => {});
   }
 
+  /**
+   * Moves an active/archived course to Recycle Bin in front-end only.
+   * This is to reduce the need to refresh the entire list of courses multiple times.
+   */
+  onDeleteFrontEnd(courseId: string): void {
```

Comment:
Naming convention:
- It is a bit weird to mention frontend here as you are already in the frontend.
- A bit redundant to say `in front-end only`
- `onXXX` is more of naming convention for an event handler.